code/modules/projectiles/ammo_datums.dm

```diff
@@ -1241,6 +1241,88 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	sundering = 1
 	damage_falloff = 0.1
 
+/datum/ammo/bullet/smarttargetrifle
+	name = "smart marksman bullet"
+	bullet_color = COLOR_SOFT_RED //Red bullets to indicate friendly fire restriction
+	hud_state = "smartgun"
+	hud_state_empty = "smartgun_empty"
+	flags_ammo_behavior = AMMO_BALLISTIC|AMMO_SUNDERING
+	damage = 40
+	penetration = 30
+	sundering = 5
+	shell_speed = 4
+	damage_falloff = 0.5
+	accurate_range = 25
+	accurate_range_min = 3
+
+/datum/ammo/bullet/spottingrifle
+	name = "smart spotting bullet"
+	bullet_color = COLOR_SOFT_RED //Red bullets to indicate friendly fire restriction
+	hud_state = "spotrifle"
+	hud_state_empty = "smartgun_empty"
+	flags_ammo_behavior = AMMO_BALLISTIC|AMMO_SUNDERING
+	damage = 70
+	penetration = 20
+	sundering = 5
+	shell_speed = 4
+	accuracy = 15
+
+/datum/ammo/bullet/spottingrifle/highimpact
+	name = "smart high-impact spotting bullet"
+	hud_state = "spotrifle_impact"
+	damage = 10
+	sundering = 0.5
+
+/datum/ammo/bullet/spottingrifle/highimpact/on_hit_mob(mob/M,obj/projectile/P)
+	staggerstun(M, P, stagger = 1, slowdown = 1)
```

Comment:
All of these have no max range override. As this is supposed to be a long range gun, I assume you want them to work past 5 tiles?

Reply:
I might slap it with a max range of 9 to 14 tiles. I want it to work past 5 but can't really decide what the max range should be beyond that. I might do 9 or 10.

Reply:
> I might slap it with a max range of 9 to 14 tiles. I want it to work past 5 but can't really decide what the max range should be beyond that. I might do 9 or 10.

Wouldn't that make it's role as a DMR pointless? 10 tiles is like 2 tiles below your screen reach.